### PR TITLE
[TASK] add `img` attr. `fetchPriority`

### PR DESCRIPTION
**Pull request by xerc**
https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/fetchPriority

### CODE REVIEW

File: tests/CommonBuilderTest.php [hunk 1]
```diff
@@ -100,8 +100,8 @@ public function isSanitizedDataProvider(): array
                 '',
             ],
             '#046' => [
-                '<img src="/typo3.org/logo.svg" alt="logo" loading="lazy" width="100" height="100" sizes="33.3vw" name="logo" align="left" border="0">',
-                '<img src="/typo3.org/logo.svg" alt="logo" loading="lazy" width="100" height="100" sizes="33.3vw" name="logo" align="left" border="0">',
+                '<img src="/typo3.org/logo.svg" alt="logo" loading="lazy" fetchPriority="low" decoding="async" width="100" height="100" sizes="33.3vw" name="logo" align="left" border="0">',
```
Comment (ohader) [inline tests/CommonBuilderTest.php:103]: seems to be used lowercased as HTML attr
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-fetchpriority

Comment (xerc) [reply tests/CommonBuilderTest.php:103]: this was my first solution but "Simon Schaufelberger" was complaining and did not response on my hint ; https://review.typo3.org/c/Packages/TYPO3.CMS/+/74926

Comment (xerc) [reply tests/CommonBuilderTest.php]: erledigt via https://review.typo3.org/c/Packages/TYPO3.CMS/+/74926/4..5

**Comment by ohader** [comment]
Thx!

**Pull request merged**